src/content/docs/alerts-applied-intelligence/applied-intelligence/anomaly-detection/automatic-anomalies.mdx

```diff
@@ -0,0 +1,26 @@
+---
+title: Automatic anomaly detection 
+tags:
+  - Alerts and applied intelligence
+  - Applied intelligence
+  - Proactive detection
+metaDescription: Learn how automatic anomaly detection in New Relic notifies you of unusual app behavior. 
+---
+
+Automatic anomaly detection is the easiest way for you and your team to detect and analyze unusual behavior in your system. It does this using the following methods:
+
+1. Anomaly detection monitors metric data reported by an agent, building a model of your typical application dynamics, and focuses on three key golden signals: throughput, response time, and errors.
+2. If one of these golden signals shows anomalous behavior, the system flags the behavior and tracks recovery to normal behavior.
+3. The system adapts to changes in your data, and continuously updates models based on new data.
+
+**Automatically on**: By default, anomaly detection monitors all your APM applications, with no action required by you. When an anomaly is detected, it's automatically surfaced in various activity streams, the applied intelligence anomalies feed and is available for querying via NRQL.
+
+**Correlation**: Anomalies are connected to our correlation logic, otherwise known as [decisions](/docs/alerts-applied-intelligence/applied-intelligence/incident-intelligence/change-applied-intelligence-correlation-logic-decisions). If our correlation logic determines that issues are connected, then a single issue will be created from multiple incidents. You can see this correlated issue in the issues feed. Anomalies provide additional context about what changed around the time an issue occurred so your team can easily detect patterns and diagnose problems. 
```

Comment:
This might be overkill, but maybe instead of using the `**Correlation**:` style, anything that's bolded and precedes a colon could be an H2. The biggest benefit I see would be breaking up the text on the page so it's more readable!